packages/teleport-component-generator-react/__tests__/integration/component-with-static-root-node.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
can you also check if these JSON files can be more easily implemented with the `uidl-builders` functions?

Reply:
yeah sure